course_content/iris_course/solutions/iris_exercise_6.1a

```diff
@@ -0,0 +1,33 @@
+
+# Iris course sample answer 6.1a
+# show what changes in a cube made by arithmetic
+
+names, cubes = ('A1B', 'E1', 'difference'), (a1b, e1, scenario_difference)
+
+print('Cube names:')
+print('    A1b  = {}'.format(a1b.name()))
+print('    E1   = {}'.format(e1.name()))
+print('    diff = {}'.format(scenario_difference.name()))
+print('result name was cleared')
```

Comment:
This is a bit strange because I get the same name for a1b and e1:
```Cube names:
    A1b  = air_temperature
    E1   = air_temperature
    diff = unknown
result name was cleared
```